integrations/linear/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@trigger.dev/linear",
+  "version": "0.0.1",
```

Comment:
Before we merge the PR, we'll need to look at the latest version of the other packages in main and set this to the same version.

We decided to have lockstep versioning for all our packages, i.e. they're always the same version as each other so compatibility and upgrading is easy to figure out. Remix do the same thing

Reply:
Good catch. Could probably grab the latest version during `create-integration`:
https://github.com/triggerdotdev/trigger.dev/blob/c2ce707f3d1d453a050dff613b4cd9fc198dd8c5/packages/cli/src/commands/createIntegration.ts#L97

..but would still have to update before merge as you mentioned.